drivers/unix/file_access_unix.h

```diff
@@ -61,6 +62,9 @@ class FileAccessUnix : public FileAccess {
 	virtual void close(); ///< close a file
 	virtual bool is_open() const; ///< true when file is open
 
+	virtual String get_path() const; /// returns the path for the current open file
```

Comment:
Shouldn't those be implemented in `FileAccessWindows` too?

Reply:
True, I'll add them